client/my-sites/checkout/get-thank-you-page-url/index.ts

```diff
@@ -693,8 +693,8 @@ function getRedirectUrlForPostCheckoutUpsell( {
 		! hasJetpackPlan( cart ) &&
 		! hasDIFMProduct( cart ) &&
 		( hasBloggerPlan( cart ) ||
-			hasPersonalPlan( cart ) ||
-			hasPremiumPlan( cart ) ||
+			hasPersonalPlan( cart ) || // if you are the person who is cleaning up hasPremiumPlan( cart ) case - consider removing this too. My rough assumption - it wasn't even used before
+			// hasPremiumPlan( cart ) || // The old promote business plan upsell page
```

Comment:
```suggestion
			// hasPremiumPlan( cart ) || // The old business plan upsell page
```

Actually shouldn't we remove that line entirely? Otherwise there is a good change that this will stay like this forever, which will make other developers wonder why when they encounter it in the future.

This makes me think, are we going to remove the [old upsell](https://github.com/Automattic/wp-calypso/pull/90284#pullrequestreview-2041028953) in another pull request? Or is it still used in some specific flows?

Reply:
As Angela mentioned here https://github.com/Automattic/dotcom-forge/issues/4187#issuecomment-2068154097 - we will review stats and then either remove the old one or revert it